x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -66,13 +44,16 @@ const HomePageComponent: React.FC<HomePageProps> = ({ children, onAppLeave }) =>
       ? SourcererScopeName.detections
       : SourcererScopeName.default
   );
-  const [showTimeline] = useShowTimeline();
 
-  const { browserFields, indexPattern, indicesExist } = useSourcererScope(
+  const { browserFields, indexPattern } = useSourcererScope(
     subPluginId.current === DETECTIONS_SUB_PLUGIN_ID
       ? SourcererScopeName.detections
       : SourcererScopeName.default
   );
+
+  // TODO: Confirm this is okay with Xavier & Patryck to replace UseUrlState
```

Comment:
@XavierM or @patrykkopycinski . I replaced the `UseUrlState` component with the below since it looked like all it was doing was running the hook. Cool with these changes?

Reply:
Ignore, undid these changes